RetailCoder.VBE/UI/Converters/SubractionConverter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Rubberduck.UI.Converters
+{
+    public class SubtractionConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var typedValue = (double)value;
+            if (!Double.TryParse((string)parameter, out var typedParam))
```

Comment:
Could this possibly be simplified to:

```
if (parameter is double typedParam)
{
    return typedValue - typedParam;
}

return typedValue;
```

Reply:
no, since parameter is a string, accordingly we need to try-parse

Reply:
Cool.